python/ctsm/run_ctsm_py_tests.py

```diff
@@ -28,6 +28,10 @@ def main(description):
         pattern = "test_unit*.py"
     elif args.sys:
         pattern = "test_sys*.py"
+    elif args.adv_unit:
```

Comment:
These changes go to the other branch and removed here.

Reply:
Looks like this should still be removed. It adds some complexity that we don't need.